src/__tests__/paxfulapi_test.ts

```diff
@@ -41,6 +41,15 @@ describe("With the Paxful API SDK", function () {
         });
     });
 
+    it('SDK points to Paxful production by default', function () {
+        process.env.PAXFUL_OAUTH_HOST = "";
```

Comment:
Is there a test that checks if all's good if the env variable is not defined at all (it is unset)? If it is unset, then "location" still should point to accounts.paxful.com

Reply:
Implementation covers it with `!process.env.PAXFUL_OAUTH_HOST` or `!process.env.PAXFUL_DATA_HOST`. But yes I can add a test for it, just that is going to be duplication of current one.